### PR TITLE
kvclient(ticdc): refine and adjust logs for better debugging

### LINKED ISSUE #6988: refine logs in the kv client

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve?

Issue Number: close #6988 

#### Tests 

 - Manual test (add detailed scripts or steps below)

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: cdc/processor/processor.go [hunk 1]
```diff
@@ -977,13 +977,6 @@ func (p *processor) createTablePipelineImpl(
 		}
 	}
 
-	log.Info("Add table pipeline", zap.Int64("tableID", tableID),
```
Comment (nongfushanquan) [inline cdc/processor/processor.go:980]: Is there any logs about adding table action already?

Comment (3AceShowHand) [reply cdc/processor/processor.go:980]: yes, it's known by other log

File: cdc/puller/ddl_puller.go [hunk 2]
```diff
@@ -597,10 +598,7 @@ func (h *ddlPullerImpl) Run(ctx context.Context) error {
 	ctx, cancel := context.WithCancel(ctx)
 	h.cancel = cancel
 
-	ctx = contextutil.PutTableInfoInCtx(ctx, -1, DDLPullerTableName)
-	ctx = contextutil.PutChangefeedIDInCtx(ctx, h.changefeedID)
```
Comment (overvenus) [inline cdc/puller/ddl_puller.go:601]: Does it cause empty changefeed id in grafana by removing this line? Could you remove `PutChangefeedIDInCtx` in `contextutil` package?

Comment (3AceShowHand) [reply cdc/puller/ddl_puller.go:601]: no, it's passed explicitly by parameters.

Comment (3AceShowHand) [reply cdc/puller/ddl_puller.go:601]: `PutChangefeedIDInCtx` cannot be removed at the moment, since it's used by other files.

**Review by sdojjy — APPROVED** [review]

**Review by overvenus — APPROVED** [review]

**Pull request merged**